public/index.html

```diff
@@ -1,9 +1,30 @@
 <html>
   <head>
     <title>タイピングゲーム</title>
-    <link rel="stylesheet" href="/stylesheets/style.css" />
-    <script type="module" src="/javascripts/main.js"></script>
+    <link rel="stylesheet" href="./stylesheets/style.css" />
+    <script type="module" src="./javascripts/main.js"></script>
   </head>
 
-  <body></body>
+  <body>
+    <br />
+    <br />
+    タイピングゲーム
+    <br />
+    <br />
+    <br />
+
+    <div class="button-container">
+      <button type="button">ゲームスタート</button>
+      <button type="button">設定</button>
+      <button type="button">クレジット</button>
+    </div>
+
+    <button class="volume-button">
+      <img src="./images/image.png" alt="ボタン画像" />
+    </button>
+
+    <button class="volume-button-on">
+      <img src="./images/FiVolume2.png" alt="ボタン画像" />
+    </button>
```

Comment:
`br`を多用せずにタイトルの位置を調整するために以下のように修正お願いします。

```suggestion
<div data-screen="title">
  <h1>タイピングゲーム</h1>
  <div class="button-container">
    <button type="button">ゲームスタート</button>
    <button type="button">設定</button>
    <button type="button">クレジット</button>
  </div>
  <button class="volume-button">
    <img src="./images/image.png" alt="ボタン画像" />
  </button>
  <button class="volume-button-on">
    <img src="./images/FiVolume2.png" alt="ボタン画像" />
  </button>
</div>
```